### PR TITLE
Add Strimzi 0.32.0 to the website

### PR DESCRIPTION
**Pull request by scholzj**
This PR adds Strimzi 0.32.0 to the Strimzi website.

### CODE REVIEW

File: _posts/2022-11-03-what-is-new-in-strimzi-0.32.0.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+author: jakub_scholz
+---
+
+Strimzi 0.32.0 has been released with multiple new features and improvements.
+As with previous releases, we created a video which introduces the main changes.
+You can watch the video here or in our YouTube channel.
+
+<!--more-->
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/q4TXUVePRJ8" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
```
Comment (ppatierno) [inline _posts/2022-11-03-what-is-new-in-strimzi-0.32.0.md:14]: adding this post was on purpose? I mean to include in this PR instead of having a new one.

**Review by ppatierno — APPROVED** [review]
LGTM. I left a question.

File: _posts/2022-11-03-what-is-new-in-strimzi-0.32.0.md (hunk 1 above)
Comment (scholzj) [reply _posts/2022-11-03-what-is-new-in-strimzi-0.32.0.md:14]: Its on purpose. Its part of the release. The reason why I did not add it in one Pr in previous releases was that I forgot about it always.

Do you see any reasons why it should not be included and should have a separate Pr instead?

Comment (ppatierno) [reply _posts/2022-11-03-what-is-new-in-strimzi-0.32.0.md:14]: It is just that I saw the blog post as separated from the operator doc, but it's not a problem at all, no concerns.

**Pull request merged**